auditlog/models.py

```diff
@@ -69,6 +69,50 @@ def log_create(self, instance, **kwargs):
             return self.create(**kwargs)
         return None
 
+    def log_m2m_changes(
+        self, changed_queryset, instance, operation, field_name, **kwargs
+    ):
+        """Create a new "changed" log entry from m2m record.
+
```

Comment:
I think you forgot to cover all params here

Reply:
You're right, thank you.